data/Yunusa_2010/metadata.yml

```diff
@@ -0,0 +1,171 @@
+    falls below 10 °C; occasionally the minimum temperature falls to 0 °C and frosts
+    occur especially from mid-winter to early spring. Summers are generally warm to
+    hot with daily mean temperature of 20.6 °C, and is generally dry except for occasional
+    heavy storms. The park was heavily grazed after the cessation of mining in the
+    early decades of the twentieth century, the declaration of the park as a reserve
+    has since alleviated grazing and facilitated recovery of the native vegetation.
+    There was limited routine harvesting of wood for fuel and timber up until 1988
+    when the practice was stopped, and there has been no record of wildfires in recent
+    decades. The park has substantial sapling undergrowth (Meers and Adams, 2003),
+    and at the commencement of this study in 2006, the dominant tree species were
+    approximately 50 years old. ^p We chose a single block of approximately one hectare
+    (62 m x 160 m) that was dominated by Eucalyptus camaldulensis (Red River Gum)
+    and E. microcarpa (Grey Box) as over-storey species. The chosen block was split
+    into two distinct zones each of approximately 0.5 ha in which the surface soils
+    have contrasting texture. A census of all young and mature trees, including the
+    shrubs, and measurement of their diameter at breast height (DBH) was undertaken
+    in May 2005. ^p Heat-pulse sensors (SF 300, Greenspan Technology, Australia) were
+    used to monitor sapflow in selected trees. Two trees of each species were instrumented
+    in the clay-zone and the sand-zone. Each tree was supplied with two probe-sets
+    consisting of a heater and a pair of thermistors that were implanted into the
+    trunk to a depth of 25 mm, after removal of the bark. The two probe heads were
+    installed on to the opposite sides of the trunks and separated by a vertical distance
+    of at least 1.5 m. The probe-sets and the area of their implantation were wrapped
+    with aluminum sheets to prevent exposure to the sun. ^p Water stored in the soil
+    profile was measured with a neutron probe (CPN, 503 DR HYDROPROBE) using pre-installed
+    aluminum access tubes to six metre depth and were spaced at 5 or 10 m intervals
+    across the two soils and were monitored mostly every fortnight, but sometimes
+    longer, intervals throughout the study period. Each soil types had 10 access tubes.
+    ^p Sapwood area and other trunk tissue parameters (wood density and wood:water ratio)
+    were measured on core samples taken from the trunk of each instrumented tree in
+    January 2007. ^p Temperature, humidity, wind speed, solar radiation and rainfall
+    were monitored with an automatic weather station installed over a short scrub
+    of vegetation about 150 m from the forest.
+  original_file: none, data transcribed from manuscript.
+  notes: Data obtained from journal article, not raw data.
+locations:
+  Reef Hills State Park, clay-zone:
```

Comment:
I'm wondering if clay zone and sand zone should be contexts rather than separate locations, but I think this works fine as well!